todo_app/app.py

```diff
@@ -205,7 +208,7 @@ def authenticate():
 
         app_user = User(github_user["id"])
         app_user.role = user_db.get_user_role(userid=github_user["id"])
-        logging.info("user is authenticated as a: %s", app_user.role)
```

Comment:
In line with submission checklist, this is an opportune place for logging user id, as there isn't anywhere else you are doing it.